packages/tabs/tab.md

```diff
@@ -47,20 +46,21 @@ The `<sp-tab>` element consists of the following:
 
 #### Icon only
 
-For tabs that have an icon and no visible label, the icon should have an `aria-label`. Icons should not be used just as decoration. If the tab item does not have a visible label, it must still have a tooltip to disclose the label.
+For tabs that have an icon and no visible label, the icon should have a `label` attribute in order to set the `aria-label` of the icon. Icons should not be used just as decoration. If the tab item does not have a visible label, it must still have a tooltip to disclose the label.
```

Comment:
Correct me if I'm wrong, my understanding of how icon works from checking the code and the documentation is that if the `label` attribute is set, it sets `aria-label` on `<sp-icon>` for us? I made a few small updates to the documentation to reflect this, but if that's wrong, please let me know.